docs/src/quickstart.md

```diff
@@ -112,8 +119,8 @@ DocTestSetup = quote
 end
 ```
 
-After the call to `JuMP.optimize!` has finished, we need to understand why the
-optimizer stopped. This can be for a number of reasons. First, the solver might
+After the call to `JuMP.optimize!` has finished, we need to query what happened.
+The solve could terminate for a number of reasons. First, the solver might
```

Comment:
Does the solve terminate, or the optimizer terminate the solve? "The optimizer can terminate the solve for a number of reasons."?

Reply:
"to terminate" (intransitive) means to come to an end (https://www.merriam-webster.com/dictionary/terminate), i.e., "the solve could come to an end for a number of reasons." Seems fine to me.